Projects/Features/Scene/GroupDetailScene/EventDetail/EventDetailView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  EventDetailView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 7/6/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import DesignSystem
+import Models
+import SwiftUI
+
+struct EventDetailView: View {
```

Comment:
```suggestion
public struct EventDetailView: View {
```
이후 만들어질 코디네이터 모듈에서 접근할 수 있도록 접근제어자를 public으로 수정해야 합니다